cuda_phase-retrieval_class/test/test_1_initial_cupy_reference/cupy_pybind.py

```diff
@@ -0,0 +1,54 @@
+import cupy as cp
```

Comment:
Is this working?

Reply:
I'm mean the complete file

Reply:
this is working, but not as a pytest. should I dekete it?

Reply:
please delete